idaplugin/rematch/actions/match.py

```diff
@@ -16,8 +16,7 @@ class MatchAction(base.BoundFileAction):
   name = "&Match"
   dialog = MatchDialog
 
-  def __init__(self, *args, **kwargs):
-    super(MatchAction, self).__init__(*args, **kwargs)
+  def reset(self):
     self._running = False
```

Comment:
I wanted to add `reset` methods, but then gave that up. Updated version no longer has that piece of code.